auth/basics/android.md

```diff
@@ -278,31 +298,35 @@ skygear.auth.loginWithUsername(username, password, object : AuthResponseHandler(
 String email = getEmail(); // get from user input
 String password = getPassword(); // get from user input
 
-skygear.getAuth().loginWithUsername(email, password, new AuthResponseHandler() {
+final String accessToken = skygear.getAuth().getCurrentAccessToken();
+
```

Comment:
I don't think the value of `accessToken` is valid because it is called evaluated and assigned before `loginWithUsername`, which is either `null` or an old access token.

Reply:
Ahh my bad. Got that fixed and pushed just now, cheers!